lib/room.cpp

```diff
@@ -1187,6 +1194,75 @@ bool Room::usesEncryption() const
     return !d->getCurrentState<EncryptionEvent>()->algorithm().isEmpty();
 }
 
+QByteArray Room::curve25519IdentityKey() const
```

Comment:
That obviously shouldn't be here, once we drop `Room` usage from `EncryptedEvent`.